### DISCUSSION TITLE
Introduction Thread

### DISCUSSION THREAD

**Discussion by MILeach** [Chaste/Chaste#25] [General]
# Introduction Thread

Come and say hello! Tell us a little bit about who you are and what work you are doing with Chaste!

**Comment by stu-l** [comment]
Hi 

I am an experienced C++ performance engineer with a bit of time on my hands.  Over the last few years I have had an occasional look at the Chaste code and have some ideas for performance improvements, but I need a little more focus.

I have looked at some of the longer running tests, but I don't know how representative they are of how Chaste is used the real world.  More particularly; the shape of the data footprint.  I'm assuming the tests ensure functionality but don't necessarily stress the hardware. 
If someone could direct me towards a 'real' example I'll get a more accurate view.  

I cant promise anything and I might get distracted onto other projects, but I can have a look and see if I can identify, and implement, anything.

Stu.
